### ISSUE TITLE
bs4

### ISSUE THREAD

**Issue by HoussemCharf** [DanMcInerney/elite-proxy-finder#11]
i am operating on raspbian the importation of the lib beautifulsoup caused some issues

"from BeautifulSoup import BeautifulSoup" change it to "from bs4 import BeautifulSoup"

just to help those are stuck there

**Comment by HoussemCharf** [comment]
also the checking isnt giving any result back :/

**Comment by DanMcInerney** [comment]
Yeah this script relies on a page that's hosted on my site danmcinerney.org. Unfortunately the site is down right now.

**Comment by erickmiller** [comment]
Maybe could you post the php scripts your server was hosting (ip-test.php , headers.php) in an etc/ dir or somewhere as part of the repo?  Thanks

**Comment by echo465** [comment]
Is this ever coming back?

**Comment by Shazgul** [comment]
@DanMcInerney Can you maybe give out the IP Test and Headers script? Or is it maybe an public script?
Found http://www.codingtheweb.com/ip-test.php that one so far, but my skills are very limited to re-write this Scanner for this Script.
